get-started/part3.md

```diff
@@ -103,6 +103,19 @@ This `docker-compose.yml` file tells Docker to do the following:
 - Define the `webnet` network with the default settings (which is a
   load-balanced overlay network).
 
+Notice that we've set the Compose file version to version 3. This essentially
+makes it [swarm mode](/engine/swarm/index.md) compatible. We can make use of the
```

Comment:
Makes sense. I'll try to tighten it up a little, not add any more detail, and see what it looks like formatted as a tip.